src/contractor/contractor.cpp

```diff
@@ -366,36 +367,56 @@ EdgeID Contractor::LoadEdgeExpandedGraph(
         throw util::exception("Limit of 255 segment speed and turn penalty files each reached");
 
     util::SimpleLogger().Write() << "Opening " << edge_based_graph_filename;
-    boost::filesystem::ifstream input_stream(edge_based_graph_filename, std::ios::binary);
-    if (!input_stream)
-        throw util::exception("Could not load edge based graph file");
+
+    const boost::interprocess::file_mapping edge_based_graph_mapping{
```

Comment:
Maybe just do

    using boost::interprocess::file_mapping;
    ...

otherwise this is super verbose and makes reading hard with our small column length.